runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/SplittableParDoNaiveBounded.java

```diff
@@ -106,8 +106,8 @@
     @Override
     public PCollectionTuple expand(PCollection<KV<byte[], KV<InputT, RestrictionT>>> input) {
       return input
-          .apply("Drop key", Values.create())
           .apply("Reshuffle", Reshuffle.of())
```

Comment:
This change was necessary to make the key coder deterministic. Note that the PCollection passed in is already using random keys.